commands/uploader.go

```diff
@@ -141,7 +143,7 @@ func verifyLocks(remote string) (ours, theirs []locking.Lock) {
 		Print("  $ git config 'lfs.%s.locksverify' true", endpoint.Url)
 	}
 
-	return ours, theirs
+	return ours, theirs, state
```

Comment:
👍. I'm not in love with the fact that `verifyLocks` returns a state, rather than taking one, but I think the refactoring to invert this isn't worth it.